content/src/Service.Host/package.json

```diff
@@ -35,42 +38,63 @@
     "babel-preset-stage-0": "^6.24.1",
     "css-loader": "^0.28.11",
     "deep-freeze": "0.0.1",
-    "enzyme": "^3.1.1",
-    "enzyme-adapter-react-16": "^1.0.4",
-    "file-loader": "^1.1.5",
-    "jest": "^21.2.1",
-    "node-sass": "^4.9.4",
+    "env-cmd": "^8.0.2",
```

Comment:
You can remove that - it's used for reading the config-test.env, for the e2e tests

Reply:
Thanks. I'll remove that